addons/web/static/src/views/form/form_controller.scss

```diff
@@ -318,8 +318,11 @@
             position: sticky;
             top: 0;
             z-index: 1;
-            background-color: $body-bg;
-            box-shadow: 0 0.3rem 0.25rem -0.25rem rgba(0, 0, 0, 0.075);
+
+            &:not(.modal .o_form_statusbar) {
```

Comment:
This selector is weird or... :thinking: 
It targets `.o_form_statusbar` that are not in a modal, right ?

Reply:
Yes that what we should target